projects/1.16.1/assets/woot/woot/lang/zh_cn.json

```
@@ -0,0 +1,535 @@
+  "guide.woot.perks.headless.1": "$(li)等级 I - 25%%几率$(li)等级 II - 50%%几率$(li)等级 III - 80%%几率",
+  "guide.woot.perks.headless.name": "从“头”开始",
+  "guide.woot.perks.looting.0": "$(item)金银岛$(0)升级插件设定生物工厂使用的抢夺等级。抢夺等级从 I 到 III。",
+  "guide.woot.perks.looting.name": "金银岛",
+  "guide.woot.perks.mass.0": "$(item)横冲直撞$(0)升级插件增加工厂生成的生物数量。更高等级生成更多数量。如果工厂包含多种生物，将使用质量最小的生物。",
+  "guide.woot.perks.mass.1": "$(li)等级 I - 2只生物$(li)等级 II - 4只生物$(li)等级 III - 6只生物",
+  "guide.woot.perks.mass.name": "横冲直撞",
+  "guide.woot.perks.rate.0": "$(item)疯狂捕食$(0)升级插件减少生物生成所需的时间。更高的等级减少更多时间。如果工厂包含多种生物，将使用速率降低最小的生物。",
+  "guide.woot.perks.rate.1": "$(li)等级 I - 20%%减少$(li)等级 II - 50%%减少$(li)等级 III - 75%%减少",
+  "guide.woot.perks.rate.name": "疯狂捕食",
+  "guide.woot.perks.slaughter.0": "这个升级插件生产工业先锋粉红史莱姆液体和肉汤于掉落物输出口旁的储罐。更高的等级提高生产的流体量。",
+  "guide.woot.perks.slaughter.1": "$(li)等级 I - 100%%$(li)等级 II - 120%%$(li)等级 III - 140%%",
+  "guide.woot.perks.slaughter.name": "翻滚血肉玩偶",
+  "guide.woot.perks.tier_shard.0": "为了制造更高等级的生物工厂，你需要使用精华碎片来制造一些方块。精华碎片只有在安装了这个升级插件的工厂才能生产。不同等级的升级插件决定碎片生成的判定次数，工厂等级决定碎片掉落几率和不同碎片种类的掉落权重。",
+  "guide.woot.perks.tier_shard.1": "$(li)等级 I - 1次精华掉落机会$(li)等级 II - 2次精华掉落机会$(li)等级 III - 3次精华掉落机会",
+  "guide.woot.perks.tier_shard.2": "等级 I 工厂拥有5.0%%几率掉落一个碎片。$(li)青瓷 - 权重 80$(li)天蓝 - 权重 20$(li)拜占庭 - 权重 1",
+  "guide.woot.perks.tier_shard.3": "等级 II 工厂拥有10.0%%几率掉落一个碎片。$(li)青瓷 - 权重 70$(li)天蓝 - 权重 25$(li)拜占庭 - 权重 4",
+  "guide.woot.perks.tier_shard.4": "等级 III 工厂拥有20.0%%几率掉落一个碎片。$(li)青瓷 - 权重 60$(li)天蓝 - 权重 30$(li)拜占庭 - 权重 8",
+  "guide.woot.perks.tier_shard.5": "等级 IV 工厂拥有30.0%%几率掉落一个碎片。$(li)青瓷 - 权重 50$(li)天蓝 - 权重 40$(li)拜占庭 - 权重 12",
+  "guide.woot.perks.tier_shard.6": "等级 V 工厂拥有40.0%%几率掉落一个碎片。$(li)青瓷 - 权重 40$(li)天蓝 - 权重 50$(li)拜占庭 - 权重 16",
+  "guide.woot.perks.tier_shard.name": "攀登高峰",
+  "guide.woot.perks.xp.0": "默认情况下，生物工厂不会生产经验。当安装了$(item)智慧$(0)升级插件，工厂将会生产$(item)经验碎片$(0)和$(item)经验尖片$(0)。更高的等级将会产出更多经验。",
+  "guide.woot.perks.xp.1": "$(li)等级 I - 100%%$(li)等级 II - 125%%$(li)等级 III - 150%%",
+  "guide.woot.perks.xp.name": "智慧",
+  "guide.woot.tools.hammer.0": "$(item)YA锤子$(0)有两个主要用途。首先是在地狱铁砧上制造物品。其次是用于原版合成。$(item)YA锤子$(0)在配方中使用永远不会被消耗。",
+  "guide.woot.tools.hammer.craft.0": "另一个锤子",
+  "guide.woot.tools.hammer.name": "YA锤子",
+  "guide.woot.tools.intern.0": "$(item)工厂搭建器$(0)是为了代替搭建和升级工厂结构这一乏味工作而诞生的。它不仅可以建造，还可以检验已有的结构而无需额外的付出。",
+  "guide.woot.tools.intern.1": "当处于任一建造模式时，$(item)工厂搭建器$(0)将从你的背包中取出工厂方块，并将其放置于世界上。建造的工厂的等级取决于$(item)工厂搭建器$(0)处于哪一模式。",
+  "guide.woot.tools.intern.2": "$(li)这个工具应在预先放置的工厂核心上使用。$(li)可以通过对着空气潜行右击来循环切换当前模式。",
+  "guide.woot.tools.intern.craft.0": "养狗时为什么要吠叫。",
+  "guide.woot.tools.intern.name": "工厂搭建器",
+  "guide.woot.tools.layout.0": "$(item)结构指导器$(0)将显示需要的方块放置于工厂的何处。它可以放置在现有工厂核心的顶部，也可以单独放置。",
+  "guide.woot.tools.layout.1": "$(li)右击以切换不同的工厂等级$(li)潜行右击以切换不同y轴层级显示",
+  "guide.woot.tools.layout.name": "结构指导器",
+  "guide.woot.tools.oracle.0": "$(item)迪的神谕$(0)允许你查看任何模拟的生物在每个抢夺等级的掉落。",
```

|Feedback Priority|
|---|
|⏳ Sand / Non-blocking but requires future consideration|

"allow"我习惯翻译为“能够”，而不是“允许”因为中文很少这么使用允许一词。

```suggestion
  "guide.woot.tools.oracle.0": "$(item)迪的神谕$(0)可让你查看在不同的抢夺等级的情况下所有模拟的生物的掉落物以及掉落几率。",
```